semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/TypeOps.scala

```diff
@@ -1,14 +1,13 @@
 package scala.meta.internal.scalacp
 
-import scala.meta.internal.{semanticdb3 => s}
-import scala.meta.internal.semanticdb3.Scala._
-import scala.meta.internal.semanticdb3.SingletonType.{Tag => st}
-import scala.meta.internal.semanticdb3.SymbolInformation.{Kind => k}
+import scala.meta.internal.{semanticdb => s}
+import scala.meta.internal.semanticdb.Scala._
+import scala.meta.internal.semanticdb.SymbolInformation.{Kind => k}
 import scala.tools.scalap.scalax.rules.scalasig._
 
 trait TypeOps { self: Scalacp =>
   implicit class XtensionTypeSType(tpe: Type) {
-    def toSemantic(linkMode: LinkMode): s.Type = {
+    def toSemanticTpe: s.Type = {
```

Comment:
This is one of the beautiful things about the Type/Signature split. We not only refine the type of `SymbolInformation.signature` (previously, `SymbolInformation.tpe`), but also gain precision in a lot of other places, e.g. here. Concretely, we observe that `toSemanticTpe` doesn't need linkMode, which is non-obvious and neat.